libnmstate/state.py

```diff
@@ -149,6 +150,31 @@ def absent(self):
         return self.state == Route.STATE_ABSENT
 
 
+class RouteRuleEntry(StateEntry):
+    def __init__(self, route_rule):
```

Comment:
A bit late to ask it and related more to the schema... but how come we do not have `state`? How will we remove it?

Reply:
Currently, route rule only support full editing:

 * If not defined in desire state, will copy from current.
 * Empty will remove all rules.
 * Non-empty will overrides.

Reply:
Ok, thanks. please mention this in the commit message (Currently full state only is supported, i.e. all rules must be mentioned when editing the route rule state).